atlas/__init__.py

```diff
@@ -1 +1,2 @@
-from atlas.generators import generator, Generator
+from .generators import generator
```

Comment:
Let's not mix import styles. Since we are doing a development install of `atlas`, it will work even if the file as a whole is run as a python script, which is useful for testing. Plus, the complete path is visible.